crates/astria-composer/src/searcher.rs

```diff
@@ -216,6 +250,18 @@ impl Searcher {
             Ok(((), ())) => {}
             Err(err) => return Err(err).wrap_err("failed to start searcher"),
         }
+
+        // set initial sequencer nonce
+        let address = Address::from_verification_key(&self.sequencer_key.verification_key());
+        let starting_nonce = self
```

Comment:
Nit (to be left for a future PR because this does not have to be addressed here): getting the nonce from the sequencer is such a strong statement that it can be used in lieu of the `abci_info` query that is used in `wait_for_sequencer`.

Reply:
agreed, i can open an issue